include/pros/imu.h

```diff
@@ -74,7 +74,7 @@ typedef struct __attribute__((__packed__)) euler_s {
  * \return 1 if the operation was successful or PROS_ERR if the operation
  * failed, setting errno.
  */
-int32_t imu_reset(uint8_t port);
+int32_t imu_calibrate(uint8_t port);
```

Comment:
```suggestion
int32_t imu_recalibrate(uint8_t port);
```
would be my suggestion since a calibration is already started for you. I'm waffling on this though. I know more then a couple people call this and I don't want to break their code, but the name is kind of bad...

Reply:
going to change this back to reset so it doesn't break user code, and changing the imu_reset function that resets all values to reset_all (or something better if you have another suggestion).